### PR TITLE
Add ens dao votes and proposals

### PR DESCRIPTION
**Pull request by soispoke**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Review by augustog — CHANGES_REQUESTED** [review]
Overall looks good, I added a few inline comments (smol change requests)

Two points:
- Since this one includes most other DAOs, should we close the others and merge this one?
- Still a bit concerned about the incremental update of vote tallies... tell me if I understood this wrong:
  - Every incremental run will calculate vote tallies every time the incremental refresh runs, and that will happen only for proposals that were created since the last run
  - If we add a row for a new proposal that is still active, and new votes roll in afterwards, will it be updated?

File: models/dao/dao_proposals.sql [hunk 1]
```diff
@@ -8,7 +8,9 @@
 }}
 
 {% set dao_proposals_models = [
-'uniswap_v3_ethereum_proposals'
+'uniswap_v3_ethereum_proposals',
```
[the remark this answers is not in the document]
Comment (soispoke) [reply models/dao/dao_proposals.sql:11]: Yes thanks for spotting! Done

File: models/dao/dao_votes.sql [hunk 2]
```diff
@@ -8,7 +8,9 @@
 }}
 
 {% set dao_votes_models = [
-'uniswap_v3_ethereum_votes'
+'uniswap_v3_ethereum_votes',
```
[the remark this answers is not in the document]
Comment (soispoke) [reply models/dao/dao_votes.sql:11]: Thanks, done as well

File: dbt_project.yml [hunk 3]
```diff
@@ -106,6 +106,20 @@ models:
         +schema: nft_optimism
         +materialized: view
 
+    compound:
```
[the remark this answers is not in the document]
Comment (soispoke) [reply dbt_project.yml:109]: 👍

**Comment by soispoke** [comment]
Closing this PR and merging the AAVE one, will reopen if @augustog still has concerns around the incremental strategy

**Comment by augustog** [comment]
No need to reopen for the sake of this, but just to clarify my concern:
- I'm not worried the merge itself will not affect past records, but rather
- Whether the subset of records we filter to update may not miss new votes

Taking the example of `ens`, if the incremental refresh fitlers the entire subset to be merged on `ENSGovernor_evt_ProposalCreated.evt_block_time > (select max(created_at) from {{ this }})`, will this not mean that only new proposals land on that subset? (=> if there's an open proposal that gets new votes, we don't catch them)

If this is wrong, please ignore my comment :)

**Pull request closed without merging**